src/components/ProgramCertificate/ProgramCertificate.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormattedDate,
+  injectIntl,
+  intlShape,
+} from '@edx/frontend-platform/i18n';
+import { Hyperlink, DropdownButton, Dropdown } from '@edx/paragon';
+import messages from './messages';
+
+function ProgramCertificate({
+  intl,
+  program_title: programTitle,
+  program_org: programOrg,
+  modified_date: modifiedDate,
+  uuid,
+  handleCreate,
+  storages = [],
+}) {
+  const showSingleAction = storages.length === 1;
+
+  const renderCreationButtons = () => (
+    <div>
+      {showSingleAction && (
+        <Hyperlink
+          className="btn btn-outline-primary"
+          onClick={() => handleCreate(uuid, storages[0].id)}
+        >
+          {intl.formatMessage(messages.certificateCardActionLabel)}
+        </Hyperlink>
+      )}
+      {!showSingleAction && (
+        <DropdownButton id="dropdown-storages" title={intl.formatMessage(messages.certificateCardMultiActionLabel)}>
+          {storages.map(({ id, name }) => (
+            <Dropdown.Item key={id} onClick={() => handleCreate(uuid, id)}>
+              {name}
+            </Dropdown.Item>
+          ))}
+        </DropdownButton>
+      )}
+    </div>
+  );
+
+  return (
+    <div className="col-12 col-sm-6 col-md-4 d-flex align-items-stretch">
+      <div className="card mb-4 certificate flex-grow-1">
+        <div className="card-body d-flex flex-column">
+          <div className="card-title">
+            <p className="small mb-0">
+              {intl.formatMessage(messages.certificateCardName)}
+            </p>
+            <h4 className="certificate-title">{programTitle}</h4>
+          </div>
+          <p className="small mb-0">
+            {intl.formatMessage(messages.certificateCardOrgLabel)}
+          </p>
+          <p className="h6 mb-4">
+            {programOrg
```

Comment:
```suggestion
            {program_org
```

Reply:
Actually, it doesn't conform current linter configuration. So I'm reverting back this.
```
Running with resolved config:
/edx/app/frontend-app-learner-record/.eslintrc.js


/edx/app/frontend-app-learner-record/src/components/ProgramCertificate/ProgramCertificate.jsx
  14:3   error  Identifier 'program_title' is not in camel case  camelcase
  15:3   error  Identifier 'program_org' is not in camel case    camelcase
  16:3   error  Identifier 'modified_date' is not in camel case  camelcase
  56:48  error  Identifier 'program_title' is not in camel case  camelcase
  62:14  error  Identifier 'program_org' is not in camel case    camelcase

/edx/app/frontend-app-learner-record/src/components/ProgramCertificateModal/ProgramCertificateModal.jsx
   18:23  error  Identifier 'app_link_android' is not in camel case  camelcase
   18:41  error  Identifier 'app_link_ios' is not in camel case      camelcase
  100:28  error  Identifier 'app_link_android' is not in camel case  camelcase
  103:28  error  Identifier 'app_link_ios' is not in camel case      camelcase
  114:19  error  Identifier 'app_link_ios' is not in camel case      camelcase
  126:19  error  Identifier 'app_link_android' is not in camel case  camelcase

✖ 11 problems (11 errors, 0 warnings)
```